### DISCUSSION TITLE
not synced

### DISCUSSION THREAD

**Discussion by umar3000** [Kale-Network/kale-blockchain#19] [General]
does not sync, what should I do?

**Comment by kaleseed** [comment]
Did you get the challenge every 10 seconds or so?

**Reply by umar3000** [reply]
did not receive

**Reply by umar3000** [reply]
![Рисунок1](https://user-images.githubusercontent.com/80649647/125223022-bb1aba80-e2d3-11eb-9297-9beeff61978e.png)

**Reply by dong3625260** [reply]
我的也是一样

**Comment by umar3000** [comment]
????

**Reply by kaleseed** [reply]
what about delete the DB file and restart the kale blockchain?
